### PR TITLE
MVP: Create a Card component.

### PR DESCRIPTION
**Pull request by AnastasiiaSorina**
[Notion.](https://www.notion.so/MVP-Create-a-Card-component-98802e7cc28c4b13bee7fc3733973090?pvs=4)

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const Card: FunctionComponent<ICardProps> = ({ data }) => {
+  return (
+    <div className="card-wrap">
+      <div className="card">
+        <h3>{data.title}</h3>
+        <p className="card-description">
+          {data.description}
+          <br />
+          See code on{" "}
+          <a className="card-link" href={data.github}>
```
Comment (TaniaZaslon) [inline src/components/Card.tsx]: Let's add `target="blank"` to links.

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

File: src/components/Card.tsx (hunk 1 above)
Comment (AnastasiiaSorina) [reply src/components/Card.tsx]: - [x] Done. [cf73870](https://github.com/AnastasiSo/Portfolio-Project/pull/22/commits/cf73870607ebc375f359fbf26865355ea07704fd)

File: src/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+const Card: FunctionComponent<ICardProps> = ({ data }) => {
+  return (
+    <div className="card-wrap">
+      <div className="card">
+        <h3>{data.title}</h3>
+        <p className="card-description">
+          {data.description}
+          <br />
+          See code on{" "}
```
Comment (TaniaZaslon) [inline src/components/Card.tsx]: Remove empty spaces {" "}.

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

File: src/components/Card.tsx (hunk 2 above)
Comment (AnastasiiaSorina) [reply src/components/Card.tsx]: - [x] Done. [a6a7f95](https://github.com/AnastasiSo/Portfolio-Project/pull/22/commits/a6a7f95d144e8c1fafcdeff06dd93a3039704425)

**Review by TaniaZaslon — APPROVED** [review]

**Pull request merged**